index.js

```diff
@@ -43,32 +45,46 @@ export default class QRCodeScanner extends Component {
     reactivateTimeout: 0,
     fadeIn: true,
     showMarker: false,
+    emptyView: <View />,
```

Comment:
Can we provide a more detailed view instead. Since this lib is focused around getting people setup and running with a QR code scanner, would be useful to add some messaging (and some basic styling perhaps) around permissions to this default `View`. Something simple with centered text should suffice.